forge-gui/res/cardsfolder/upcoming/the_swarmlord.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Swarmlord
+ManaCost:3 G U R
+Types:Legendary Creature Tyranid
+PT:5/5
+K:etbCounter:P1P1:X::Rapid Regeneration — CARDNAME enters the battlefield with two +1/+1 counters on it for each time you've cast your commander from the command zone this game.
+SVar:X:Count$TotalCommanderCastFromCommandZone/Twice
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.YouCtrl+HasCounters | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Xenos Cunning — Whenever a creature you control with a counter on it dies, draw a card.
+SVar:TrigDraw:DB$ Draw | NumCards$ 1
+DeckHas:Ability$Counters
```

Comment:
Maybe also `DeckHInts:Ability$Counters` since this benefits when other things have counters (and die) too?